### PR TITLE
Rating added in CL

### PR DESCRIPTION
**Pull request by humzahasan**

### CODE REVIEW

File: library/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+.stars input {
+  display: none;
+}
+
+.stars label {
+  float: right;
+  font-size: 4rem;
+  color: var(--white-color);
+  margin: 0 0.5rem;
```
Comment (zoyatanweer) [inline library/rating/rating.css:14]: use pixels in place of 0.5rem.

**Review by zoyatanweer — APPROVED** [review]
clean code, good to merge.

File: library/rating/rating.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.stars {
+  height: 10rem;
+  /* width: 20rem; */
```
Comment (purohitdheeraj) [inline library/rating/rating.css]: commented code can be removed

File: library/rating/rating.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+}
+.stars input {
+  display: none;
+}
+.stars label {
+  float: right;
+  font-size: 4rem;
+  color: #ffffff;
+  margin: 0 0.5rem;
+  text-shadow: 2px 2px 2px #fc8019;
```
Comment (purohitdheeraj) [inline library/rating/rating.css]: var() can be used over here for color

**Review by purohitdheeraj — APPROVED** [review]
Good to merge with minor changes for color (var)

File: library/rating/rating.css (hunk 1 above)
Comment (humzahasan) [reply library/rating/rating.css:14]: Have been using the same throughout the project, reason being i use 16px = 1rem as base. So here my calculation is easy with 0.5rem = 8px.

File: library/rating/rating.css (hunk 3 above)
Comment (humzahasan) [reply library/rating/rating.css]: Implementing in the next PR.

**Pull request merged**